### ISSUE TITLE
The performance when testing on very small dataset

### ISSUE THREAD

**Issue by XuqianRen** [graphdeco-inria/gaussian-splatting#377]
Hi,

Thank you for your excellent work. When I tested Gaussian splatting with my own small volume datasets that captured one object and only contained around 40 frames in the dataset, I found the rendered test frame rendered badly. Will this be caused by the current parameters not fitting the small volume dataset? If yes, what kinds of parameters can I adjust to fit these kinds of dataset? Thank you for your consideration.

**Comment by wkailiu** [comment]
I also encountered this problem, do you know how to solve this problem? @XuqianRen

**Comment by XuqianRen** [comment]
No, I haven't. But I think the problem is mainly caused by the blurry and sparse input in my dataset. What does the data look like in your dataset? @wkailiu
